Source/Generator/README.md

```diff
@@ -3,53 +3,40 @@
 This repo contains the LabVIEW Measurement template code
 
 ## Dependency List
+
 1. LabVIEW 2020 64 bit
-2. [LabVIEW gRPC Support](https://github.com/ni/grpc-labview/releases)
-This repo contains the support for LabVIEW gRPC. Download and install the two latest release packages for,
-2.1 ni_lib_labview_grpc_library-x.x.x.x.vip
-2.2 ni_lib_labview_grpc_servicer-x.x.x.x.vip
+2. [LabVIEW gRPC Support](https://github.com/ni/grpc-labview/releases) This repo contains the support for LabVIEW gRPC. Download and install the two latest release packages for
+    1. ni_lib_labview_grpc_library-x.x.x.x.vip
+    2. ni_lib_labview_grpc_servicer-x.x.x.x.vip
 
 ## Developer Guide
 
 ### Instantiate a New Measurement
 
 1. Create a new LabVIEW project (say `Measurement.lvproj`) and open it.
-2. Open the New Measurement Instantiation window, from the project window, goto `Tools` -> `TAF` -> `Measurement Plugin` -> `New...`
-2.1 In the New window, Enter the `Measurement Service Name` that you would like to give to the measurement and click `Create Measurement Service`.
-2.2 This will create a new measurement service library in the project.
+2. Open the New Measurement Instantiation window, from the project window, goto `Tools` -> `MeasurementLink` -> `Create Measurement Service...`
+    1. In the New window, Enter the `Measurement Service Name` that you would like to give to the measurement and click `Create Measurement Service`.
+    2. This will create a new measurement service library in the project.
```

Comment:
```suggestion
2. From the project window, select `Tools` -> `MeasurementLink` -> `Create Measurement Plug-in...`
3. In the dialog that appears, enter the `Measurement Plug-in Name` that you would like to give to the measurement, then click `Create Measurement Plug-in`. This will create a new measurement service plug-in library in the project.
```

Reply:
Updated the new measurement dialog screenshot. Let me know if there's other screenshots that need updating.